config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
@grahamwhaley we are not there yet... all of our patches are not yet upstream. Also once they make it upstream the distros should pick them up.  @anthonyzxu will have the latest upstream status.